setup.py

```diff
@@ -11,56 +10,19 @@
     "or remotely from ImJoy.io"
 )
 
-try:
-    subprocess.call(["conda", "-V"])
-    if os.name == "nt":
-        # for fixing CondaHTTPError:
-        # https://github.com/conda/conda/issues/6064#issuecomment-458389796
-        process = subprocess.Popen(
-            ["conda", "info", "--json", "-s"], stdout=subprocess.PIPE
-        )
-        cout, err = process.communicate()
-        conda_prefix = json.loads(cout.decode("ascii"))["conda_prefix"]
-        print("Found conda environment: {}".format(conda_prefix))
-        os.environ["PATH"] = (
-            os.path.join(conda_prefix, "Library", "bin")
-            + os.pathsep
-            + os.environ["PATH"]
-        )
-except OSError:
-    if sys.version_info > (3, 0):
-        print(
-            "WARNING: you are running ImJoy without conda, "
-            "you may have problem with some plugins"
-        )
-    else:
-        sys.exit(
-            "Sorry, ImJoy Python Plugin Engine can only run within a conda environment "
-            "or at least in Python 3."
-        )
+requirements = ["python-socketio[client]>=4.1.0", "numpy"]
```

Comment:
We should pin this. An upgrade could be breaking.

Reply:
Module level variables should be all capital letters.

Reply:
Done, thanks!